package.json

```diff
@@ -30,36 +31,39 @@
   },
   "homepage": "https://github.com/babel/babel-library-boilerplate",
   "devDependencies": {
-    "babel": "^5.0.0",
-    "babelify": "^6.0.0",
-    "browserify": "^8.1.1",
-    "chai": "^2.0.0",
+    "babel": "^5.2.17",
```

Comment:
I know this. But as I stated earlier I get trouble with changing version numbers on all of the dependencies because of nodejJS 12.2. It's require a lot of different versions. At least in my end. So I skipped that part.

For the main files as you pointed out I did this just to try to be compatible with all nodeJS versions. As you are running v. 10 and v. 12 in test cases. But I will remove the operator.